assets/components/organisms/event/event-just-finished.twig

```diff
@@ -1 +1,8 @@
-<img src="./images/wip-screenshot/event-just-finished.png" class="img-fluid" />
+{% extends "@organisms/event/event.twig" %}
+
+{% block classes %}card card-grayscale link-trapeze-horizontal bg-gray-100{% endblock %}
```

Comment:
Should it be also switched to `.card-gray`? Impacts the background-color on hover.

Reply:
Not swtiched, but work on a better correlation 😉